src/httpserver/utilities/arguments.clj

```diff
@@ -0,0 +1,12 @@
+(ns httpserver.utilities.arguments
+  (:require [clojure.tools.cli :refer [parse-opts]]))
+
+(def cli-options
+  [["-p" "--port PORT"
+    :default 8080
+    :parse-fn #(Integer/parseInt %)]
+   ["-d" "--directory DIRECTORY"
+    :default  "./"]])
+
+(defn buildMap [args]
```

Comment:
Use kebab-case instead of camelCase for, well, just about everything in Clojure (except maybe things that interop directly with Java).

Also, this is a pretty vague name for a public function, especially considering almost everything in Clojure is a map.